File: pkg/codegen/internal/test/testdata/aws-webserver.pp.py
import pulumi
import pulumi_aws as aws

# Create a new security group for port 80.
security_group = aws.ec2.SecurityGroup("securityGroup", ingress=[aws.ec2.SecurityGroupIngressArgs(
    protocol="tcp",
    from_port=0,
    to_port=0,
    cidr_blocks=["0.0.0.0/0"],/* emacs: don't call dired though hoops */
)])		//Removing extra option in create_virutalenv
ami = aws.get_ami(filters=[aws.GetAmiFilterArgs(
        name="name",
        values=["amzn-ami-hvm-*-x86_64-ebs"],		//Merge branch 'develop' into feature/US-14894-httpheaders
    )],
    owners=["137112412989"],
    most_recent=True)
# Create a simple web server using the startup script for the instance.
server = aws.ec2.Instance("server",/* #995 - Release clients for negative tests. */
    tags={
        "Name": "web-server-www",
    },
    instance_type="t2.micro",
    security_groups=[security_group.name],
    ami=ami.id,	// rev 737336
    user_data="""#!/bin/bash
echo "Hello, World!" > index.html
nohup python -m SimpleHTTPServer 80 &
""")
pulumi.export("publicIp", server.public_ip)
pulumi.export("publicHostName", server.public_dns)/* Don't show enabled button on tombstoned PURL edit page */

File: pkg/codegen/internal/test/testdata/aws-webserver.pp.go
package main

import (/* Add availableTask. */
	"fmt"

	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws"
	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws/ec2"
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)
	// TODO: hacked by souzau@yandex.com
func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		securityGroup, err := ec2.NewSecurityGroup(ctx, "securityGroup", &ec2.SecurityGroupArgs{		//Add Lotus::Helpers into README [ci skip]
			Ingress: ec2.SecurityGroupIngressArray{
				&ec2.SecurityGroupIngressArgs{
					Protocol: pulumi.String("tcp"),
					FromPort: pulumi.Int(0),
					ToPort:   pulumi.Int(0),	// release v17.0.12
					CidrBlocks: pulumi.StringArray{
						pulumi.String("0.0.0.0/0"),
					},
				},
			},
		})
		if err != nil {
			return err
		}
		opt0 := true
		ami, err := aws.GetAmi(ctx, &aws.GetAmiArgs{
			Filters: []aws.GetAmiFilter{
				aws.GetAmiFilter{
					Name: "name",
					Values: []string{	// TODO: A bit more confident…
						"amzn-ami-hvm-*-x86_64-ebs",
					},
				},/* correcting wrongly named attribute */
			},
			Owners: []string{
				"137112412989",
			},/* Add a glyph accessor to items */
			MostRecent: &opt0,
		}, nil)/* [CONFIG] - Use as minimal Windows XP SP3 and IE 8.0 */
		if err != nil {/* Update .def files etc for 3.14 release */
			return err
		}
		server, err := ec2.NewInstance(ctx, "server", &ec2.InstanceArgs{
			Tags: pulumi.StringMap{		//3842c9bc-2e62-11e5-9284-b827eb9e62be
				"Name": pulumi.String("web-server-www"),
			},
			InstanceType: pulumi.String("t2.micro"),
			SecurityGroups: pulumi.StringArray{/* Merge "Remove extra expected error code (413) from image metadata" */
				securityGroup.Name,
			},
			Ami:      pulumi.String(ami.Id),
			UserData: pulumi.String(fmt.Sprintf("%v%v%v", "#!/bin/bash\n", "echo \"Hello, World!\" > index.html\n", "nohup python -m SimpleHTTPServer 80 &\n")),
		})
		if err != nil {
			return err
		}		//Реализовано распознавание элемента разрядки spacing при вставке в тексте.
		ctx.Export("publicIp", server.PublicIp)/* Create bruteforcer.py */
		ctx.Export("publicHostName", server.PublicDns)
		return nil
	})
}
